css/src/components/chip.css

```diff
@@ -0,0 +1,126 @@
+.Chip {
+    display: flex;
+    box-sizing: border-box;
+    border-radius: var( --spacing-l);
+    padding-right: var( --spacing);
+    padding-left: var( --spacing);
+    padding-top: var( --spacing-s);
+    padding-bottom: var( --spacing-s);
+    width: fit-content;
+    justify-content: space-between;
+    border-style: solid;
+    border-width: var( --spacing-xs);
+    align-items: center;
+    flex-direction: row;
+    
+  }
+  
+  .Chip-wrapper {
+    display: flex;
+    align-items: center;
+  }
+  
+  .Chiplabel {
+    font-size: var(--font-size);
+    font-family: Nunito Sans;
+    text-align: left;
+    line-height: var(--font-height);
+  }
+  
+  .Chip-icon--left {
+    padding-right: var( --spacing-m);
+    display: flex;
+    align-items: center;
+    color: var(--night);
+  }
+  
+  .Chip-icon--right {
+    padding-left: var( --spacing-m);
+    display: flex;
+    align-items: center;
+    color: var(--night-light);
+  }
+  
+  .Chip--Action {
+    background: var( --shadow-0);
+    border-color: var(--stone);
+    font-weight: var( --font-weight-normal);
+    cursor: pointer;
+  }
+  
+  .Chip--Action:hover {
+    background: var(--stone-light);
+    border-color: var(--stone);
+    cursor: pointer;
+  }
+  
+  .Chip--Action:focus {
+    background: var(--stone-lighter);
+    cursor: pointer;
+  }
+  
+  .Chip--Action:active {
+    background: var(--stone-dark);
+    cursor: pointer;
+  }
+  
+  .Chip--disabled--Action {
+    background: var(--shadow-0);
+    color: var(--night-lightest);
+    border-color: var(--stone-light);
```

Comment:
Add default cursor for disabled state and remove color

Reply:
Also follow camelCase naming convention for modifiers too.